gammapy/stats/variability.py

```diff
@@ -289,9 +289,12 @@ def TimmerKonig_lightcurve_simulator(
     power_spectrum,
     npoints,
     spacing,
-    leakage_protection=1.0,
+    nchunks=10,
     random_state="random-seed",
     power_spectrum_params=None,
+    mean=0.0,
+    std=1.0,
+    poisson=False,
```

Comment:
Does it always apply? What is it supposed to do for flux like quantities?

Reply:
You are right, in the case of flux quantities the result of the poisson noise will probably be 0: the option is expected to be used with "norm"-like arrays. Added a warning for the case